CHANGES.rst

```diff
@@ -13,6 +13,12 @@ ipac.nexsci.nasa_exoplanet_archive
 
 - Fixed InvalidTableError for DI_STARS_EXEP and TD tables. [#3189]
 
+simbad
+^^^^^^
+
+- [BREAKING] The detailed hierarchy is now returned by default in ``query_hierarchy``
```

Comment:
maybe we should have a new API changes section? Having `[BREAKING]` is very unusual

Reply:
I was trying to make API changes more obvious to avoid comment like we got in the last issue about the version numbers. A new section would be a nice place to put this, yes.

Reply:
ok, then could you add an 

```
API changes
-----------
````

heading when you rebase this?

Reply:
Done :)
And the PR is ready for review